### PR TITLE
SC-486 Correct callable V4 results

### PR DESCRIPTION
**Pull request by xrtm000**

### CODE REVIEW

File: node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala [hunk 1]
```diff
@@ -72,26 +73,62 @@ case class InvokeScriptTrace(
...
+      "description"  -> issue.description,
+      "isReissuable" -> issue.isReissuable,
+      "quantity"     -> issue.quantity,
+      "name"         -> issue.name
+    )
+
+  private def reissueJson(reissue: Reissue) =
+    Json.obj(
+      "assetId"      -> reissue.assetId.toString,
+      "isReissuable" -> reissue.isReissuable,
```
Comment (msmolyakov) [inline node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala]: rename to "reissuable" as field Asset.reissuable and as in json

File: node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala [hunk 2]
```diff
@@ -72,26 +73,62 @@ case class InvokeScriptTrace(
...
+      "decimals"     -> issue.decimals,
+      "description"  -> issue.description,
+      "isReissuable" -> issue.isReissuable,
+      "quantity"     -> issue.quantity,
+      "name"         -> issue.name
+    )
+
+  private def reissueJson(reissue: Reissue) =
+    Json.obj(
+      "assetId"      -> reissue.assetId.toString,
```
Comment (msmolyakov) [inline node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala]: is needed to rename to "id" as field `Asset.id`? How do you think?

File: node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala [hunk 3]
```diff
@@ -72,26 +73,62 @@ case class InvokeScriptTrace(
...
+      "isReissuable" -> reissue.isReissuable,
+      "quantity"     -> reissue.quantity
+    )
+
+  private def burnJson(burn: Burn) =
+    Json.obj(
+      "assetId"  -> burn.assetId.toString,
```
Comment (msmolyakov) [inline node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala]: is needed to rename to "id" as field `Asset.id`? How do you think?

File: node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala (hunk 2 above)
Comment (xrtm000) [reply node/src/main/scala/com/wavesplatform/transaction/smart/script/trace/TraceStep.scala]: I would not change to make it clearer

**Pull request merged**